lib/events/athena/consumer.go

```diff
@@ -284,12 +284,8 @@ func runWithMinInterval(ctx context.Context, fn func(context.Context) bool, minI
 func (c *consumer) processBatchOfEvents(ctx context.Context) (reachedMaxSize bool, e error) {
 	start := time.Now()
 	var size int
-	// TODO(tobiaszheller): we need some metrics to track it.
-	// And that log message should be deleted.
 	defer func() {
-		if size > 0 {
-			c.logger.Debugf("Batch of %d messages processed in %s", size, time.Since(start))
-		}
+		consumerBatchProcessingDuration.Observe(time.Since(start).Seconds())
```

Comment:
Just to ensure. Do we want to include events collection time to the  "BatchProcessing duration" ?

Reply:
yes